OptimizelySDK.Tests/AudienceConditionsTests/ConditionEvaluationTest.cs

```diff
@@ -302,6 +309,38 @@ public void TestGTMatcherReturnsTrueWhenAttributeValueIsGreaterThanConditionValu
         }
 
         #endregion // GTMatcher Tests
+        
+        #region GEMatcher Tests
+
+        [Test]
+        public void TestGEMatcherReturnsFalseWhenAttributeValueIsLessButTrueForEqualToConditionValue()
+        {
+            Assert.That(GECondition.Evaluate(null, new UserAttributes { { "distance_ge", 5 } }, Logger), Is.False);
```

Comment:
use `Assert.IsTrue or Assert.IsFalse`

Reply:
Evaluate return bool? and Assert.IsTrue expects bool. That is the reason I used Assert.That